icubam/db/synchronizer.py

```diff
@@ -113,6 +118,26 @@ def sync_users(self, users_df, force_update=False):
         except Exception as e:
           logging.error("Cannot add user to icu: {}. Skipping".format(e))
 
+  def sync_bed_counts(self, bedcounts_df):
+    raise NotImplementedError("WIP")
+    bedcounts_df = bedcounts_df[BC_COLUMNS]
+
+    # First check that all ICUs exist:
+    icu_names = set(bedcounts_df['icu_name'].unique())
+    db_icus = dict((icu.name, icu) for icu in self.db.get_icus())
+
+    # Make sure each bedcount has an existent ICU:
+    icu_diff = icu_names - set(db_icus.keys())
+    if icu_diff is not None:
+      KeyError("Missing ICUs in DB: {icu_diff}. Please add them first.")
+
+    # Now we are sure all ICUs are present so we can insert without checking:
+    for bc in bedcounts_df.iterrows():
+      item = None
+      self.db.update_bed_count_for_icu(
+        None, store.BedCount(**item), force=True
```

Comment:
I believe instead you could also pass a user to this (these) function. Since this might be used from the backoffice where there is user calling this function. Hence if would do
`self.db.update_bed_count_for_icu(user, store.BedCount(**item), force=user is None)`

Reply:
I added something that should work, let me know if it's good (once I push).